### DISCUSSION TITLE
Subtitles not display correctly

### DISCUSSION THREAD

**Discussion by jackthenewbie** [AlistGo/alist#3057] [Q&A] [answered]
I'm using ``xhofe/alist:latest`` image. 
![image](https://user-images.githubusercontent.com/55600733/212573611-ffe459ac-b5c4-43b6-b5ac-1253afa1ada8.png)
Subtitles is .vtt

How to reproduce:
Choose a video has auto-generate subtitles. Download it with ``yt-dlp --write-auto-subs [youtube-link]``

Is this a bug or I set something incorrectly?

**Comment by NovaShen555** [comment, marked as answer]
It seems to support SRT instead of VTT, so convert VTT to SRT.
Hava nice day!

**Reply by jackthenewbie** [reply]
Not necessary an answer, but a work around. Thanks:)

**Reply by NovaShen555** [reply]
like to online converter
https://www.veed.io/tools/subtitle-converter/vtt-to-srt
